FAQ.md

```diff
@@ -64,6 +64,6 @@ This is expected. Due to randomness of self-play games, once Black choose to pas
 ## 对局结果错误 ##
```

Comment:
If you had a second could you also like issue #198 in line 62, and change defailed to detailed